dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/metrics/MetricsTestServlet.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.reactive.messaging.fat.kafka.metrics;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Emitter;
+
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.KafkaTestConstants;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.AbstractKafkaTestServlet;
+
+import jakarta.inject.Inject;
+import jakarta.servlet.annotation.WebServlet;
+
+/**
+ *
+ */
+@SuppressWarnings("serial")
+@WebServlet("/MetricsTest")
+public class MetricsTestServlet extends AbstractKafkaTestServlet {
+
+    public static final String EMITTER_OUTGOING_CHANNEL = "MetricsEmitter";
+    public static final String EMITTER_TOPIC = "EmitterTopic";
+    public static final String OUTGOING_TOPIC = "OutgoingTopic";
```

Comment:
More a note for the future than something that _needs_ changed: try to keep your names consistent.

For each test you have a test method, a method in the servlet, two channel names, two constant names, one or two helper beans and a topic name. Ideally, these should all be named consistently so that it's obvious at a glance which test it belongs to.

I know that these tests have gone through several revisions, and that's probably where the inconsistency has come from, but it's often worth going back through at the end and making everything match up. It will make it easier to read and debug later.